flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/streaming/util/TestStreamEnvironment.java

```diff
@@ -95,6 +96,14 @@ private static void randomize(Configuration conf) {
             final PseudoRandomValueSelector valueSelector =
                     PseudoRandomValueSelector.create(testName != null ? testName : "unknown");
             valueSelector.select(conf, ExecutionCheckpointingOptions.ENABLE_UNALIGNED, true, false);
+            valueSelector.select(
+                    conf,
+                    ExecutionCheckpointingOptions.ALIGNMENT_TIMEOUT,
+                    Duration.ofSeconds(0),
+                    Duration.ofSeconds(1),
+                    Duration.ofSeconds(5),
+                    Duration.ofSeconds(30),
```

Comment:
Are three of those settings really necessary? Maybe we can stick to `0`, short value (100ms) and long value (1s? 2s?)? At least until it's proven that we really need to test more cases?